tests/test_endpoints.py

```diff
@@ -0,0 +1,79 @@
+import pytest
+import json
+from app import create_app
+import sys
+sys.path.append("..")
+
+
+@pytest.yield_fixture
+def app():
+    app = create_app("testing")
+    yield app
+
+
+@pytest.fixture
+def test_cli(loop, app, sanic_client):
+    return loop.run_until_complete(sanic_client(app))
+
+
+async def test_index(test_cli):
+    resp = await test_cli.get('/')
+    assert resp.status == 200
+
+
+async def test_getLanguages_with_no_query_param(test_cli):
+    resp = await test_cli.get('/languages')
+    resp_json = await resp.json()
+    assert resp_json["total_results"] == 0
+    assert resp_json["page"] == 1
+    assert resp.status == 200
+
+
+async def test_getLanguages_with_more_query_param(test_cli):
+    params = {'page': 2, 'type': 'local', 'limit': 50}
+    resp = await test_cli.get('/languages', data=json.dumps(params))
+    resp_json = await resp.json()
+    assert resp_json["message"] == "Expecting less than 3 arguments"
+    assert resp.status == 400
+
+
+async def test_getLanguages_with_valid_query_param(test_cli):
+    params = {'page': 2, 'type': 'local'}
+    resp = await test_cli.get('/languages', data=json.dumps(params))
+    resp_json = await resp.json()
+    assert resp_json["total_results"] == 0
```

Comment:
Need to test the actual data. you might need to create some record in the test database of use mocking to pretend you send something to the database. 
While writing tests make sure they are Isolated, fast. I encourage you to read the following as it provides good insights into the quality of great unit tests https://www.kenneth-truyers.net/2012/12/15/key-qualities-of-a-good-unit-test/

Reply:
I will refactor by adding dummy data for tests